validator/client/validator_attest.go

```diff
@@ -202,6 +176,63 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		trace.StringAttribute("bitfield", fmt.Sprintf("%#x", aggregationBitfield)),
 	)
 }
+func (v *validator) preSigningValidations(ctx context.Context, indexedAtt *ethpb.IndexedAttestation, pubKey [48]byte) error {
+	fmtKey := fmt.Sprintf("%#x", pubKey[:])
+	log := log.WithField("pubKey", fmt.Sprintf("%#x", bytesutil.Trunc(pubKey[:]))).WithField("slot", indexedAtt.Data.Slot)
+
+	v.attesterHistoryByPubKeyLock.RLock()
+	attesterHistory := v.attesterHistoryByPubKey[pubKey]
+	v.attesterHistoryByPubKeyLock.RUnlock()
+	if featureconfig.Get().ProtectAttester {
```

Comment:
These feature flags are confusing. (`ProtectAttester` and `SlasherProtection`) Can we rename it to something `local` / `remote` ?

Reply:
@0xKiwi do you think of better naming?

Reply:
Maybe we should unify the local protection flags in another PR, I'm not sure about `remote` protection though since it is specifically through the slasher, not as general as `remote` sounds.

Reply:
lets do it in another pr. @terencechain WDYT?